client/blocks/login/continue-as-user.scss

```diff
@@ -1,34 +1,68 @@
 .continue-as-user {
-	display: flex;
+	display: block;
 	animation: 1s appear 0s ease-in;
-	align-items: center;
-	justify-content: center;
-	flex-direction: column;
 	color: var( --color-text-subtle );
 	font-size: $font-body-small;
 	text-align: center;
+	height: calc( 70vh - 80px );
```

Comment:
This seems a bit made-up, can we add a comment as to where it came from?

Reply:
You are right. I am not really sure exactly why those numbers have been chosen but they make things look okay across different screen sizes that I have tested.